src/main/java/se/sundsvall/disturbance/api/model/Category.java

```diff
@@ -4,11 +4,36 @@
 
 @Schema(description = "Category model", enumAsRef = true)
 public enum Category {
-	COMMUNICATION,
-	DISTRICT_COOLING,
-	DISTRICT_HEATING,
-	ELECTRICITY,
-	ELECTRICITY_TRADE,
-	WASTE_MANAGEMENT,
-	WATER;
+
+	COMMUNICATION("COMMUNICATION"),
+	DISTRICT_COOLING("DISTRICT_COOLING"),
+	DISTRICT_HEATING("DISTRICT_HEATING"),
+	ELECTRICITY("ELECTRICITY"),
+	ELECTRICITY_TRADE("ELECTRICITY_TRADE"),
+	WASTE_MANAGEMENT("WASTE_MANAGEMENT"),
+	WATER("WATER");
+
+	private final String value;
```

Comment:
Since the values are the same as the actual enum value, you could use Category.name() instead in the `fromValue` method.

Reply:
Absolutely, was thinking of some future obscure `SOME_NEW_CATEGORY("THIS CATEGORY HAS SPACES IN IT")`, then our current enum wouldn't work :)
But for now, you're correct. Keep or throw away?